Client/Frontend/Home/BookmarksPanel.swift

```diff
@@ -23,7 +23,7 @@ class BookmarksPanel: SiteTableViewController, HomePanel {
     weak var homePanelDelegate: HomePanelDelegate? = nil
     var source: BookmarksModel?
     var parentFolders = [BookmarkFolder]()
-    var bookmarkFolder = BookmarkRoots.MobileFolderGUID
+    var bookmarkFolder: BookmarkFolder?
```

Comment:
That would simplify things, though there are a couple obstacles: neither `BookmarksFolderTitleMobile` nor `BookmarkFolder`'s initializer are public.